Hyper/Meshes/Chunk.cs

```diff
@@ -25,8 +25,14 @@ internal class Chunk : Mesh
 
     private TypedIndex _shape;
 
+    private BepuPhysics.Collidables.Mesh? collisionSurface;
```

Comment:
is there a way to get rid of nullable values? They generally make the code less robust

Reply:
i mean we can, i could use something similar to createdVertexArrayObject flag, which isn't great-looking. give me a hint if you know a professional way of dealing with such problems